### ISSUE TITLE
Please add new 2.0 tracks

### ISSUE THREAD

**Issue by RedstoneWizard08** [4321ba/Wynncraft_Noteblock_OST#2]
Hi! I would like the new Wynncraft v2.0 tracks to be added to this repo since it's a great resource for getting MIDI and NBS files of the OST, which I absolutely love. Thanks!

**Comment by 4321ba** [comment]
Yup! thats definitely something that could be done! though I'm currently not super interested in this project, so I'll probably not record music in the near future, but there were people recording stuff, the only wc2.0 music I currently have can be downloaded from a google drive link under this video:
https://www.youtube.com/watch?v=L7TTUkqprQ0
or here directly: https://drive.google.com/drive/folders/1XT2FvBJ4gGifAtcPHCYoc81UEgiPg2Ha
but thanks for showing interest :DD

**Comment by RedstoneWizard08** [comment]
Alright, thanks!

**Comment by 4321ba** [comment]
I mean we can keep this open, as it's not resolved yet
